parallel-install/pkg/helm/client.go

```diff
@@ -17,22 +21,46 @@ import (
 
 const logPrefix = "[helm/client.go]"
 
+//Config provides configuration for the Client
 type Config struct {
-	HelmTimeoutSeconds            int
-	BackoffInitialIntervalSeconds int
-	BackoffMaxElapsedTimeSeconds  int
-	Log                           func(format string, v ...interface{})
+	HelmTimeoutSeconds            int                                   //Underlying native Helm client processing timeout
+	BackoffInitialIntervalSeconds int                                   //Initial interval for exponential-backoff retry algorithm
+	BackoffMaxElapsedTimeSeconds  int                                   //Maximum time for exponential-backoff retry algorithm
+	Log                           func(format string, v ...interface{}) //Used for logging
 }
 
+//Client type implements ClientInterface
```

Comment:
```suggestion
//"Client type" implements ClientInterface.
```

Reply:
Changed to: `//Client implements the ClientInterface.`